common/flavorization/wc_title_holders.txt

```diff
@@ -86,128 +127,1878 @@ warlord_female = {
+}
+emperor_feudal_female_nightelf = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = 400
+	top_liege = no
+	governments = { feudal_government theocracy_government }
+	cultures = { night_elf }
+}
+empire_feudal_nightelf = {
+	type = title
+	tier = empire
+	priority = 400
+    governments = { feudal_government theocracy_government }
+	cultures = { night_elf }
+}
+
+### Highborne / Nightbornes ###
+# Nighborne specific
+count_feudal_male_nightborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government }
+	cultures = { nightborne }
+}
+count_feudal_female_nightborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = county
+	priority = @county_cultural_prio
+	governments = { feudal_government }
+	cultures = { nightborne }
+}
+county_feudal_nightborne = {
+	type = title
+	tier = county
+	priority = @county_cultural_prio
+	cultures = { nightborne }
     governments = { feudal_government }
 }
 
-# Duke
-prince_nightborne_male = {
+# Feudal
+duke_feudal_male_highborne = {
 	type = character
 	gender = male
 	special = holder
 	tier = duchy
-	priority = @duchy_cultural_prio
+	priority = 400
 	top_liege = no
 	governments = { feudal_government }
 	cultures = { nightborne highborne }
 }
-princess_nightborne_female = {
+duke_feudal_female_highborne = {
 	type = character
 	gender = female
 	special = holder
 	tier = duchy
-	priority = @duchy_cultural_prio
+	priority = 400
 	top_liege = no
 	governments = { feudal_government }
 	cultures = { nightborne highborne }
 }
-# Duchy
-principality_nightborne = {
+duchy_feudal_highborne = {
 	type = title
 	tier = duchy
-	priority = @duchy_cultural_prio
+	priority = 400
+	cultures = { nightborne highborne }
+    governments = { feudal_government }
+}
+king_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = kingdom
+	priority = 400
+	governments = { feudal_government }
+	cultures = { nightborne highborne }
+}
+king_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = kingdom
+	priority = 400
+	governments = { feudal_government }
+	cultures = { nightborne highborne }
+}
+kingdom_feudal_highborne = {
+	type = title
+	tier = kingdom
+	priority = 400
 	cultures = { nightborne highborne }
     governments = { feudal_government }
+}
+emperor_feudal_male_highborne = {
+	type = character
+	gender = male
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	governments = { feudal_government }
+	cultures = { nightborne highborne }
+}
+emperor_feudal_female_highborne = {
+	type = character
+	gender = female
+	special = holder
+	tier = empire
+	priority = @empire_cultural_prio
+	governments = { feudal_government }
+	cultures = { nightborne highborne }
+}
+
+### Nagas ###
+count_feudal_male_naga = {
+	type = character
+	gender = male
+	special = holder
+	tier = county
+	priority = 400
```

Comment:
Why you don't use local values? `@empire_cultural_prio`, `@kingdom_cultural_prio`, etc.

Reply:
done in 357957c6f

Reply:
The most of the titles don't use the values still.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/123699893-d36ced00-d870-11eb-9937-bb9f4b3e96bd.png)

</details>